dev/change_version.py

```diff
@@ -0,0 +1,142 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This script also needs to change the versions found in `AssemblyInfo.cs` as well as the NuGet versions.